q2_feature_table/_summarize/summarize_assets/app/js/child.js

```diff
@@ -0,0 +1,12 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) 2016-2017, QIIME 2 development team.
+//
+// Distributed under the terms of the Modified BSD License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+// ----------------------------------------------------------------------------
+
+
+document.addEventListener('DOMContentLoaded', function() {
+  parent.postMessage(document.body.scrollHeight, '*');
```

Comment:
[Get a real browser!](http://stackoverflow.com/questions/15932650/body-scrollheight-doesnt-work-in-firefox), haha that is really weird though, as it works as expected for me on macOS Firefox 50.1.0. I'll add those fallbacks to it and see if that helps for Ubuntu.